src/main/java/org/folio/marccat/dao/persistence/AuthorityNote.java

```diff
@@ -1,11 +1,18 @@
 package org.folio.marccat.dao.persistence;
 
+import org.folio.marccat.dao.SystemNextNumberDAO;
+import org.folio.marccat.shared.CorrelationValues;
+import org.folio.marccat.util.StringText;
+
+import net.sf.hibernate.HibernateException;
+import net.sf.hibernate.Session;
+
 /**
  * @author elena
```

Comment:
Generally speaking, if you're going to have a javadoc comment block, it should contain a description

Reply:
I have changed it.